draft-ietf-taps-interface.md

```diff
@@ -1048,7 +1048,10 @@ candidate transport-layer connections to be created to the specified remote
 endpoint. The caller may immediately begin sending Messages on the Connection
 (see {{sending}}) after calling Initate(); note that any idempotent data sent
 while the Connection is being established may be sent multiple times or on
-multiple candidates.
+multiple candidates. 
+Also note that transport-layer connections that match the original
+Selection Properties may automatically be created at any later time,
```

Comment:
This is a little too open/scary— don't say that they may automatically be created at any time, that sounds like there's no way to opt out, etc. Do we need a comment here in the API? This section isn't about transport layer stuff, just about initiate.

Reply:
I don't think this is a wise place to dig: saying transport connections can be opened later is not something I would like to see in TAPS (Maybe I share the view this COULD happen, but I'd really like to avoid saying anyting in this topic at all in the API.

Reply:
+1

Reply:
I would like to give implementations this kind of freedom, e.g., to automatically re-initiate a  transport connection after the server closed an old, idle one. But maybe this hint fits better into implementation.

Reply:
Yes, I think implementations can have the freedom to do many things without the need to put it in the base spec.